### PR TITLE
Improve Asset loading Warning Issue #482

### PR DESCRIPTION
**Pull request by bernardthered**

### CODE REVIEW

**Review by pathunstrom — APPROVED** [review]

**Review by AstraLuma — CHANGES_REQUESTED** [review]
If I'm reading this correctly, this will add this warning text for every kind of asset with a `file_missing` behavior, but the text is only applicable to `ppb.Image`.

**Comment by bernardthered** [comment]
@AstraLuma good catch! I corrected that issue, let me know what you think of the solution, I'd be happy to change anything about that :-)

**Comment by bernardthered** [comment]
@pathunstrom it looks like another spurious unit test error (this time the error is "❗ Agent is not responding!"). Would you be able to press the rerun link when you get a chance?

File: ppb/assetlib.py [hunk 1]
```diff
@@ -165,6 +165,10 @@ class AbstractAsset(abc.ABC):
     This defines the common interface for virtual assets, proxy assets, and
     real/file assets.
     """
+    # Can be overridden on child classes to add an additional message to show after "File not
+    # found: %r.". See Image.not_found_message for an example.
```
Comment (AstraLuma) [inline ppb/assetlib.py]: This should be #: so it gets picked up by sphinx

**Review by AstraLuma — CHANGES_REQUESTED** [review]
Minor changes. Otherwise, I'm happy with this design and implementation.

File: ppb/assetlib.py (hunk 1 above)
Comment (bernardthered) [reply ppb/assetlib.py]: I didn't know about that #: syntax, thanks for the pointer. I just read a bit about it and updated those two comment lines.

**Comment by bernardthered** [comment]
I also updated the warning message to be more explicit, let me know if you think the original is better and I can change it back.

**Review by AstraLuma — APPROVED** [review]
bors r+

**Pull request merged**